Sources/Thumbprint/Components/Dropdown.swift

```diff
@@ -189,7 +189,11 @@ public final class Dropdown: Control, UIContentSizeCategoryAdjusting {
     @objc
     private func pickerViewDidShow(notification: Notification) {
         if isFirstResponder, selectedIndex == nil {
-            selectedIndex = pickerView.selectedRow(inComponent: 0)
+            let numberOfRowsInPicker = pickerView.numberOfRows(inComponent: 0)
+            let selectedRow = pickerView.selectedRow(inComponent: 0)
+            if selectedRow < numberOfRowsInPicker, selectedRow >= 0 {
```

Comment:
This is fine but you can also do it via range condition ([article](https://www.hackingwithswift.com/articles/90/how-to-check-whether-a-value-is-inside-a-range)).
`if (0..<numberOfRowsInPicker) ~= selectedRow { }`
or more readable
`if (0..<numberOfRowsInPicker).contains(selectedRow) { }`